jmix-reports/reports-flowui/src/main/java/io/jmix/reportsflowui/view/parameter/ReportParameterDetailView.java

```diff
@@ -363,6 +366,23 @@ protected void initMetaClassLookup() {
         ComponentUtils.setItemsMap(metaClassField, metaClassesOptionsMap);
     }
 
+    @Subscribe
+    public void onValidation(final ValidationEvent event) {
+        boolean isValidated = getContent().getComponents().stream().noneMatch(component ->
+                component instanceof SupportsValidation<?>
```

Comment:
`viewValidation.validateUiComponents(getContent())` method not working?

Reply:
Still doesn't check not enabled components